consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -27,13 +39,31 @@ class StreamsSinkProcedures {
         }
 
         val properties = config?.mapValues { it.value.toString() } ?: emptyMap()
-        val configuration = StreamsSinkProcedures.streamsEventSinkConfigMapper.convert(config = properties)
+        val configuration = streamsEventSinkConfigMapper.convert(config = properties)
 
         val data = readData(topic, config ?: emptyMap(), configuration)
 
         return data.map { StreamResult(mapOf("data" to it)) }.stream()
     }
 
+    @Procedure("streams.sink.start")
+    fun sinkStart(): Unit {
+        checkEnabled()
+        streamsEventSink?.start()
+    }
+
+    @Procedure("streams.sink.stop")
+    fun sinkStop(): Unit {
+        checkEnabled()
+        streamsEventSink?.stop()
+    }
+
+    @Procedure("streams.sink.restart")
```

Comment:
return the start operation result

Reply:
done, but if there is an error in the stop I'll return it with the error